src/py/flwr/simulation/run_simulation.py

```diff
@@ -297,20 +297,22 @@ def _run_simulation(
     """
     if backend_config is None:
         backend_config = {}
+        backend_config["init_args"] = {}
 
     # Set logging level
     logger = logging.getLogger("flwr")
     if verbose_logging:
         update_console_handler(level=DEBUG, timestamps=True, colored=True)
     else:
-        backend_config["silent"] = True
+        backend_config["init_args"]["logging_level"] = WARNING
+        backend_config["init_args"]["log_to_driver"] = False
```

Comment:
Although not ideal, it's better to leave it as `True`. Otherwise by default nothing will be printed/logged when a user adds a simple print statement to the client (e.g. in `<client>.fit()`). Or do you think it's better to leave it as `False` and then introduce two levels of "verbosity" as a flag for `flower-simulation`. Currently if you pass `--verbose` it shows all the `DEBUG` logs. Maybe one idea is to have a `--client-verbose` flag that simply sets `log_to_driver=True`? what do you think?